programs/zstd.1

```diff
@@ -165,7 +165,7 @@ Additionally, this can be used to limit memory for dictionary training\. This pa
 \fB\-f\fR, \fB\-\-force\fR: disable input and output checks\. Allows overwriting existing files, input from console, output to stdout, operating on links, block devices, etc\.
 .
 .IP "\(bu" 4
-\fB\-c\fR, \fB\-\-stdout\fR: write to standard output (even if it is the console)
```

Comment:
We typically do our `man` modifications into `zstd.1.md`, which then is used to automatically generate `zstd.1`.
I recommend you ported your doc change onto `zstd.1.md`, otherwise it will be lost next time `zstd.1` is regenerated.

Reply:
good catch, I did not notice that. moved the change to .md and regenerated .1.